### PR TITLE
Link download button to latest release

### PR DESCRIPTION
**Pull request by samgranger**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
           <section class="col-50 hero__content">
             <h1>Capture your screen</h1>
             <h2>An open-source screen recorder built with web technology.</h2>
-            <a href="https://github.com/wulkano/kap/releases/download/v2.0.0/Kap-2.0.0.dmg" class="button hero__button">
+            <a href="https://github.com/wulkano/kap/releases/download/v2.1.1/Kap-2.1.1.dmg" class="button hero__button">
```
Comment (albinekb) [inline src/index.html:66]: I think we should link to http://kap-updates.now.sh/download instead so it's always the latest

Comment (albinekb) [reply src/index.html:66]: But I think this is fine for now as the new website is almost done 💅 👍

**Comment by davidjb** [comment]
Found the same issue -- `brew cask install` installs a later version than the one linked to from the homepage.  Had me scratching my head.

**Comment by skllcrn** [comment]
Closing in favour of #18

**Pull request closed without merging**